### PR TITLE
fix: optimize bind-i18n styles

### PR DESCRIPTION
**Pull request by gene9831**
English | [简体中文](https://github.com/opentiny/tiny-engine/blob/develop/.github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.zh-CN.md)

# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-engine/blob/develop/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Built its own designer, fully self-validated

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] Other... Please describe: Styles

## Background and solution

优化属性面板绑定国际化UI

### What is the current behavior?

Issue Number: N/A

### What is the new behavior?

效果图

1. 选择不带参数的国际化文案
{46C028F1-6827-494D-8447-9D43AAA1619B}

2. 选择带参数的国际化文案
{CC555796-5B80-465D-82ED-6B65B4AB4541}

3. 添加国际化并关联
{49B390A8-7946-46F6-A020-E328D921C977}

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

遗留问题：在 `tiny-select` 组件的下拉面板中，固定了面板宽度后，如果某个选项的文本长度超长，会变成可横向滚动的面板（见下图1）。在 `tiny-vue` 最新版中，会自动截去多余文本，并且悬停可以显示详情（见下图2）。建议升级 `tiny-vue` 版本来解决这个问题，否则需要侵入式修改部分style

图1：下拉面板可横向滚动
image

图2：`tiny-vue` 最新版选择器
image

## Summary by CodeRabbit

- **New Features**
	- Updated placeholder text for the language selection dropdown.
	- Added a title to the popover component for improved user guidance.

- **Improvements**
	- Enhanced layout and styling for buttons and dropdowns, improving visual consistency.
	- Increased width of the popover for better content display.
	- Introduced new styles for button alignment and spacing.

- **Bug Fixes**
	- Adjusted button types for consistency across the interface.

### CODE REVIEW

File: packages/common/component/BindI18n.vue [hunk 1]
```diff
@@ -17,18 +18,19 @@
         </tiny-option>
       </tiny-select>
       <div v-if="paramsForm.length" class="params-form">
-        <label>国际化参数配置</label>
+        <div class="label">国际化参数配置</div>
```
[the remark this answers is not in the document]
Comment (gene9831) [reply packages/common/component/BindI18n.vue:17]: `label` element is a inline-level element. I need block-level enelent in this place

File: packages/common/component/I18nInput.vue [hunk 2]
```diff
@@ -124,9 +125,24 @@ export default {
...
+    position: absolute;
+    top: 0;
+    right: 0;
   }
 }
 </style>
+
+<style>
+.i18n-input-popover {
+  padding: 20px !important;
```
Comment (chilingling) [inline packages/common/component/I18nInput.vue]: 看看能不能不用 `!important`

File: packages/common/component/BindI18n.vue [hunk 3]
```diff
@@ -209,23 +198,37 @@ export default {
...
     align-items: center;
     label {
       width: 80px;
     }
-    margin-bottom: 10px;
   }
 }
+.bottom-buttons {
+  display: flex;
+  flex-direction: row-reverse;
```
Comment (chilingling) [inline packages/common/component/BindI18n.vue]: 这里为什么要使用 `row-reverse` 来调换顺序？直接在 template 调用顺序可以嘛？

Comment (gene9831) [reply packages/common/component/BindI18n.vue]: 这里需要实现按钮靠右的效果。`float` 和 `text-align` 属性有些不合适，使用了 `row-reverse` 来实现靠右，但是 `row-reverse`是反转布局，写代码时不直观。这里找到了更合适的 `justify-content: flex-end` 来实现靠右的效果

**Review by hexqi — DISMISSED** [review]

File: packages/common/component/I18nInput.vue (hunk 2 above)
Comment (gene9831) [reply packages/common/component/I18nInput.vue]: 由于全局的css选择器链路很长，优先级特别高，改成了`.tiny-popover.tiny-popover.tiny-popper[x-placement].i18n-input-popover`才能覆盖全局样式

**Review by chilingling — APPROVED** [review]

**Pull request merged**